src/h5.h

```diff
@@ -94,7 +94,10 @@ template <typename T> class h5d_chunked_1d {
   h5d_chunked_1d();
```

Comment:
I know it isn't in your changes, but this constructor is not actually implemented.